database/index.js

```diff
@@ -0,0 +1,39 @@
+const mongoose = require('mongoose');
+mongoose.connect('mongodb://localhost/gamedb');
```

Comment:
Should lines 2 through 9 and line 26 be included within the getGame function?

Reply:
Had irl discussion with John and Zane about said topic. Seems to be good to go.